operations/validate_csm_health.md

```diff
@@ -408,54 +166,45 @@ Known failures and how to recover:
-
-There are multiple [Goss](https://github.com/aelsabbahy/goss) test suites available that cover a variety of sub-systems.
-
-Run the NCN health checks against the three different types of nodes with the following commands:
 
-**IMPORTANT:** These tests may only be successful while booted into the PIT node. Do not run these as part of upgrade testing. This includes the Kubernetes check in the next block.
+<a name="pet-optional-ncnhealthchecks-resources"></a>
+### 1.2 OPTIONAL Check of ncnHealthChecks Resources
 
+To dump the ncn uptimes, the node resource consumptions and/or the list of pods not in a running state, run the following:
 
 ```bash
-pit# /opt/cray/tests/install/ncn/automated/ncn-healthcheck-master
-pit# /opt/cray/tests/install/ncn/automated/ncn-healthcheck-worker
-pit# /opt/cray/tests/install/ncn/automated/ncn-healthcheck-storage
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s ncn_uptimes
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s node_resource_consumption
+ncn# /opt/cray/platform-utils/ncnHealthCheck.sh -s pods_not_running
 ```
+<a name="known-issues"></a>
+#### 1.2.1 Known Issues
 
-And the Kubernetes test suite via:
+* pods_not_running
+  - If the output of `pods_not_running` indicates that there are pods in the `Evicted` state, it may be due to the /root file system being filled up on the Kubernetes node in question. Kubernetes will begin evicting pods once the root file system space is at 85% until it is back under 80%. This may commonly happen on ncn-m001 as it is a location that install and doc files may be downloaded to. It may be necessary to clean up space in the /root directory if this is the root cause of pod evictions. The following commands can be used to determine if analysis of files under /root is needed to free-up space.
 
-```bash
-pit# /opt/cray/tests/install/ncn/automated/ncn-kubernetes-checks
-```
+    ```bash
+    ncn# df -h /root
+    Filesystem      Size  Used Avail Use% Mounted on
+    LiveOS_rootfs   280G  245G   35G  88% /
+    ```
 
-<a name="autogoss-issues"></a>
-#### 1.8.1 Known Test Issues
+    ```bash
+    ncn# du -h -s /root/
+    225G  /root/
+    ```
 
-* These tests can only reliably be executed from the PIT node. Should be addressed in a future release.
-* K8S Test: Kubernetes Query BSS Cloud-init for ca-certs
-  - May fail immediately after platform install. Should pass after the TrustedCerts Operator has updated BSS (Global cloud-init meta) with CA certificates.
-* K8S Test: Kubernetes Velero No Failed Backups
-  - Because of a [known issue](https://github.com/vmware-tanzu/velero/issues/1980) with Velero, a backup may be attempted immediately upon the deployment of a backup schedule (for example, vault). It may be necessary to use the `velero` command to delete backups from a Kubernetes node to clear this situation.
+    ```bash
+    ncn# du -ah -B 1024M /root | sort -n -r | head -n 10
+    ```
+  - The `cray-crus-` pod is expected to be in the Init state until slurm and munge
```

Comment:
"Slurm" should be capitalized it's a company's product.

Reply:
MUNGE should be in all caps: https://dun.github.io/munge/

Reply:
done